examples/phoenix_app/mix.exs

```diff
@@ -46,6 +46,7 @@ defmodule PhoenixApp.Mixfile do
       {:phoenix, "~> 1.4"},
       {:plug_cowboy, "~> 2.0"},
       {:jason, "~> 1.0"},
+      {:httpoison, "~> 1.7"},
```

Comment:
I just found out this version specification can cause problems with projects that aren't up to the latest version of HTTPoison. This will work better:

```elixir
{:httpoison, "~> 1.0"}
```